modules/gdscript/gdscript_editor.cpp

```diff
@@ -620,8 +620,8 @@ static String _trim_parent_class(const String &p_class, const String &p_base_cla
 	}
 	Vector<String> names = p_class.split(".", false, 1);
 	if (names.size() == 2) {
-		String first = names[0];
-		String rest = names[1];
+		const String &first = names[0];
+		const String &rest = names[1];
 		if (ClassDB::class_exists(p_base_class) && ClassDB::class_exists(first) && ClassDB::is_parent_class(p_base_class, first)) {
 			return rest;
```

Comment:
```suggestion
		if (ClassDB::class_exists(p_base_class) && ClassDB::class_exists(first) && ClassDB::is_parent_class(p_base_class, first)) {
			return names[1];
```

Reply:
I'm unsure about removing the assignment here. I decided to move `rest` into the `if` statement  instead.